views.go

```diff
@@ -546,7 +546,7 @@ func (lk *logKeeper) findLogs(ctx context.Context, query bson.M, sort bson.D, mi
 			return
 		}
 
-		for cur.Next(ctx) {
+		for cur.Next(db.Context()) {
```

Comment:
Do we still need to pass in a context to the function?

Reply:
I think this is the same discussion as the previous thread. Let's continue the discussion there and I'll change both if we decide they need to change.